gallery/src/components/NxTabs/NxTabsModalExample.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxModal,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import {faAngry} from '@fortawesome/free-solid-svg-icons';
+
+const NxTabsModalExample = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [activeTabId, setActiveTabId] = useState<number>(0);
+
+  function handleTabSelect(index: number) {
+    setActiveTabId(index);
+  }
+
+  return (
+    <>
+      <button onClick={() => setShowModal(true)} className="nx-btn">Open Modal</button>
+      { showModal &&
+        <NxModal id="nx-modal-simple-example" onClose={modalCloseHandler} className="nx-modal--wide">
```

Comment:
Unnecessary class, I don't think this really needs to be a wide modal

Reply:
> Unnecessary class, I don't think this really needs to be a wide modal

If it isn't then the four tabs can't display - which goes to your comment below and depending on what @dturner32 says I might make this modal normal to show the behavior

Reply:
> > Unnecessary class, I don't think this really needs to be a wide modal
> 
> If it isn't then the four tabs can't display - which goes to your comment below and depending on what @dturner32 says I might make this modal normal to show the behavior

Now with flex we don't need to worry as much about the modal width, removed `--wide` here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/e8558e498d966f6300fc18dfd12eb12c04172191